datasets/CodeCompliantEquipment.idf

```diff
@@ -0,0 +1,2427 @@
+    40.0,                     !- Maximum Value of y
+    ,                         !- Minimum Curve Output
+    ;                         !- Maximum Curve Output
+
+Curve:Biquadratic,
+    ASHRAE901_2019_air_A_lt150ton_10.1kWpton_13.7IPLV.IP_cap-f-t,    !- Name
+    0.7536274941282223,       !- Coefficient1 Constant
+    0.059276028733216114,     !- Coefficient2 x
+    -0.002073292464305097,    !- Coefficient3 x2
+    0.00831633652983885,      !- Coefficient4 y
+    -0.00026246704098650017,    !- Coefficient5 y2
+    -0.00011269975583401686,    !- Coefficient6 xy
+    4.0,                      !- Minimum Value of x
+    15.6,                     !- Maximum Value of x
+    10.0,                     !- Minimum Value of y
+    40.0,                     !- Maximum Value of y
+    ,                         !- Minimum Curve Output
+    ;                         !- Maximum Curve Output
+
+Curve:Biquadratic,
+    ASHRAE901_2019_air_A_gte150ton_10.1kWpton_14IPLV.IP_eir-f-t,    !- Name
+    0.872388053480053,        !- Coefficient1 Constant
+    -0.02372502054976646,     !- Coefficient2 x
+    0.0011453381173352607,    !- Coefficient3 x2
+    -0.013271002349552512,    !- Coefficient4 y
+    0.0006189144448899488,    !- Coefficient5 y2
+    -0.00025179743901172335,    !- Coefficient6 xy
+    4.0,                      !- Minimum Value of x
+    15.6,                     !- Maximum Value of x
+    10.0,                     !- Minimum Value of y
+    40.0,                     !- Maximum Value of y
+    ,                         !- Minimum Curve Output
+    ;                         !- Maximum Curve Output
+
+Curve:Biquadratic,
+    ASHRAE901_2019_air_A_gte150ton_10.1kWpton_14IPLV.IP_cap-f-t,    !- Name
+    0.7869057566821718,       !- Coefficient1 Constant
+    0.06561212364217481,      !- Coefficient2 x
+    -0.002301768917046292,    !- Coefficient3 x2
+    0.00697010050343958,      !- Coefficient4 y
+    -0.00026792045664183377,    !- Coefficient5 y2
+    -0.00016228919441438013,    !- Coefficient6 xy
+    4.0,                      !- Minimum Value of x
+    15.6,                     !- Maximum Value of x
+    10.0,                     !- Minimum Value of y
+    40.0,                     !- Maximum Value of y
+    ,                         !- Minimum Curve Output
+    ;                         !- Maximum Curve Output
+
+Curve:Biquadratic,
+    ASHRAE901_2019_air_A_lt528kW_2.985COP_4.048IPLV.SI_eir-f-t,    !- Name
+    0.8870774118142857,       !- Coefficient1 Constant
+    -0.024855155962524283,    !- Coefficient2 x
+    0.0011930867281970867,    !- Coefficient3 x2
+    -0.013818136673476294,    !- Coefficient4 y
+    0.0006361097011564334,    !- Coefficient5 y2
+    -0.0002740904902173277,    !- Coefficient6 xy
+    4.0,                      !- Minimum Value of x
+    15.6,                     !- Maximum Value of x
+    10.0,                     !- Minimum Value of y
+    40.0,                     !- Maximum Value of y
```

Comment:
Likewise, these inputs will be constrained even if the loop temps or setpoints go outside this range. Chilled water temps 4-15.6C (39.2-60.08 F) and condenser water temps of 10-40C (50-104 F). Are these ranges adequate?

Reply:
We believe so. We could re-generate them with extended temperature ranges if need be.